app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,92 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+$gray-medium-light: #eaeaea;
```

Comment:
Please please this color variable into a new file and then import it from this file.